cassandra-diagnostics-ft/basic/pom.xml

```diff
@@ -72,15 +72,12 @@
                                         <wait>
                                             <time>120000</time>
                                             <log>Starting listening for CQL clients</log>
-                                            <exec>
-                                                <postStart>sleep 60s</postStart>
```

Comment:
This pause was introduced as it turned out that even the message `Starting listening for CQL clients` is present in the cassandra log file, it is actually not ready to accept client requests. Though, it might be the case that the latest version of cassandra and respective docker image changed that behavior.